.env.template

```diff
@@ -4,3 +4,9 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
+
+TRELLO_API_KEY='0e8c5313a51ec05165d5846ccdb0b984'
+TRELLO_API_SECRET='02436356bfd15cc68db990ef94cc1f07ac2eca4526a7c3bc672201a588ea7879'
+TRELLO_BOARD_ID='60a21e48ae8d9a3a66965e3e'
```

Comment:
These are real credentials and should not be checked in to our source code! We should remove these, but also treat these as compromised and rotate these credentials

We should only specify the key or a placeholder value in the `.env.template`, and allow new developers to fill in the relevant value